manifests/scripts.pp

```diff
@@ -0,0 +1,16 @@
+# @summary Deploy helper scripts scripts
+#
+# @api private
+#
+class letsencrypt::scripts () {
+  assert_private()
+
+  # required in letsencrypt::certonly
+  file { '/usr/local/sbin/letsencrypt-domain-validation':
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0500',
+    source => "puppet:///modules/${module_name}/domain-validation.sh",
```

Comment:
while you update this, please use content to inline the file into the catalog. that reduces the number of http calls from puppet agent to puppetserver.
```suggestion
    content => file("${module_name}/domain-validation.sh"),
```